### PR TITLE
Removes a few spawns

### PR DESCRIPTION
**Pull request by ChangelingRain**
cut it out

### CODE REVIEW

File: code/modules/power/antimatter/shielding.dm [hunk 1]
```diff
@@ -28,11 +28,9 @@
...
 
 
-/obj/machinery/am_shielding/New(loc)
-	..(loc)
-	spawn(10)
-		controllerscan()
-	return
+/obj/machinery/am_shielding/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, .proc/controllerscan, 1), 10)
```
Comment (XDTM) [inline code/modules/power/antimatter/shielding.dm]: shouldn't it be 0?

Comment (Cyberboss) [inline code/modules/power/antimatter/shielding.dm]: There shouldn't be a param at all

Comment (ChangelingRain) [inline code/modules/power/antimatter/shielding.dm]: don't ask me, I just converted it.

Comment (Cyberboss) [inline code/modules/power/antimatter/shielding.dm]: the original had no parameter, you're calling it with the param `1`

**Pull request merged**